homeassistant/components/smartthings/config_flow.py

```diff
@@ -118,58 +121,76 @@ async def async_step_user(self, user_input=None):
             _LOGGER.exception(
                 "API error setting up the SmartApp: %s", ex.raw_error_response
             )
-            return self._show_step_user(errors)
+            return self._show_step_pat(errors)
         except ClientResponseError as ex:
             if ex.status == 401:
                 errors[CONF_ACCESS_TOKEN] = "token_unauthorized"
+                _LOGGER.exception("Unauthorized error received setting up SmartApp")
```

Comment:
We shouldn't log these as `exception`. Only use `exception` if it's an unexpected error (so `except Exception`). Is it needed to still log it as an error since we're showing the result to the user?

Reply:
I changed it to debug but still log the exception as it includes a correlation ID that support can use to figure out why the error occurred.